platform/windows/windows_utils.cpp

```diff
@@ -155,7 +155,11 @@ Error WindowsUtils::copy_and_rename_pdb(const String &p_dll_path) {
 	} else if (!FileAccess::exists(copy_pdb_path)) {
 		copy_pdb_path = dll_base_dir.path_join(copy_pdb_path.get_file());
 	}
-	ERR_FAIL_COND_V_MSG(!FileAccess::exists(copy_pdb_path), FAILED, vformat("File '%s' does not exist.", copy_pdb_path));
+	if (!FileAccess::exists(copy_pdb_path)) {
+		// The PDB file may be distributed separately on purpose, so we don't consider this an error.
+		WARN_VERBOSE(vformat("File '%s' does not exist.", copy_pdb_path));
```

Comment:
I would maybe go so far as to say that this shouldn't even be emitted as a verbose warning, since there isn't really anything particularly weird or erroneous happening, but given the amount of noise in the `--verbose` output already I'm not sure there's much harm in it.

Perhaps some more context could be given at least, since there's none given when you return `ERR_SKIP`:

```suggestion
		WARN_VERBOSE(vformat("PDB file '%s' for library '%s' was not found, skipping copy/rename.", copy_pdb_path, p_dll_path));
```

Reply:
I like having the verbose message, in case we ever need to debug something going wrong with the PDB copying.

However, I agree about adding some more context! I've updated the message for your suggestion in my latest push